lib/zipkin-tracer/config.rb

```diff
@@ -39,6 +39,10 @@ def initialize(app, config_hash)
       if @sampled_as_boolean
         @logger && @logger.warn("Using a boolean in the Sampled header is deprecated. Consider setting sampled_as_boolean to false")
       end
+      # Record the given tags on server receive, even if the zipkin headers were present in the incoming request?
+      tags = config[:record_on_server_receive]
+      @record_on_server_receive = present?(tags) ? parse_record_on_server_receive(tags) : nil
```

Comment:
the reason I went this way is that the standard case is the user doesn't have this option set _and_ the service is called by another zipkin-enabled one. In this 99% of the time case, we don't need to do anything at all, e.g.

- `zipkin_env.called_with_zipkin_headers?` is true 
- `@config.record_on_server_receive` is falsy 
=> `tags` var is nil 
=> `trace_request_information` doesn't do anything

If I go the default empty hash route here I'll either have to check for nil and empty or I'll have to create a new hash object every time just so that `.each` doesn't blow.